go.mod

```diff
@@ -0,0 +1,88 @@
+module vcr-cli
```

Comment:
set the module name to name of repo since it is a public module, this will allow people to `go get` the code github.com/Vonage/vonage-cloud-runtime-cli